src/utils/viewing.js

```diff
@@ -60,27 +60,27 @@ function renderViewingActivityList(viewedItems) {
   const list = document.getElementById('viewingActivityList');
   for (const viewedItem of viewedItems) {
     const date = document.createElement('div');
-    date.classList = ['col ns-date-col'];
+    date.classList = ['col date nowrap'];
     date.innerText = viewedItem.dateFormatted;
 
     const title = document.createElement('a');
     title.href = `/title/${viewedItem.movieID}`;
     title.innerText = viewedItem.title;
 
     const titleContainer = document.createElement('div');
-    titleContainer.classList = ['col ns-title-col'];
+    titleContainer.classList = ['col title'];
     titleContainer.appendChild(title);
 
     const duration = document.createElement('div');
-    duration.classList = ['col ns-duration-col'];
+    duration.classList = ['col report'];
     duration.innerText = viewedItem.durationFormatted;
 
     const type = document.createElement('i');
-    type.classList = ['material-icons ns-icon'];
-    type.innerText = viewedItem.type === 'Movie' ? 'local_movies' : 'tv';
+    type.classList = ['material-icons ns-icon-list'];
+    type.innerText = viewedItem.type === 'Película' ? 'local_movies' : 'tv';
```

Comment:
You caught a bug here, it seems that the correct icon was not shown in Spanish because of the incorrect matching of the **_viewedItem.type_** with the hardcoded English word **_Movie_**. 

Anyway, it should work for both English and Spanish translations, so I think the best thing to do here is to compare the **_viewedItem.type_** against the translation of the word **_movie_**:

```js
type.innerText = viewedItem.type === `${chrome.i18n.getMessage('movie')}` ? 'local_movies' : 'tv';
```

I will change it as it's a minor change, so you don't have to update your  PR